src/mqueryfront/src/query/QueryMatchesItem.js

```diff
@@ -22,7 +26,11 @@ const QueryMatchesItem = (props) => {
         ));
 
     const matchBadges = Object.values(matches).map((v) => (
-        <span key={v} className="badge badge-pill badge-primary ml-1 mt-1">
+        <span
+            key={v}
+            className="badge badge-pill badge-primary ml-1 mt-1 cursor-pointer"
+            onClick={() => addFilter(v)}
```

Comment:
Arrow function calling another arrow function with wrong argument name (suggesting that we're dealing with event when it's a match). It should be simplified.
```suggestion
            onClick={() => props.changeFilter(v)}
```